loyal/chain.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "loyal",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.joinloyal.com/",
+  "pretty_name": "Loyal",
+  "chain_id": "loyal-1",
+  "bech32_prefix": "loyal",
```

Comment:
please register this prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done & merged! https://github.com/satoshilabs/slips/pull/1440